manifests/build-support/mkSpagoDerivation.nix

```diff
@@ -1,40 +1,11 @@
-{
-  lib,
-  stdenv,
-  fetchurl,
-  nodejs,
-}: {
-  version,
-  url,
-  hash,
-}:
-stdenv.mkDerivation rec {
+{ lib }:
+import ./mkDerivation.nix {
   pname = "spago";
-  inherit version;
-
-  src = fetchurl {inherit url hash;};
-
-  nativeBuildInputs = [nodejs];
-
-  buildPhase = ''
-    tar xf $src
-  '';
-
-  installPhase = ''
-    mkdir -p $out/bin
-
-    mv package/bin/bundle.js $out/bin/bundle.js
-    mv package/package.json $out/package.json
-
-    chmod +x $out/bin/bundle.js
-    patchShebangs $out/bin/bundle.js
-
-    ln -s $out/bin/bundle.js $out/bin/spago
-  '';
-
-  meta = with lib; {
+  exe = "spago";
+  js = "bin/bundle.js";
```

Comment:
I named this `bundle.js` but there's really no obligation; it can be `bin/index.js` like the others, so we could just remove the `js` key and make them all `index.js`

Reply:
Then, `purs-backend-es` sources should also have `bin/index.js`.